src/net.cpp

```diff
@@ -2123,6 +2122,8 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
 {
     AssertLockNotHeld(m_total_bytes_sent_mutex);
 
+    auto now = GetTime<std::chrono::microseconds>();
```

Comment:
`GetTime()` is deprecated.

```suggestion
    const auto now = NodeClock::now();
```

Reply:
This works:

```diff
@@ -2125 +2125 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
-    auto now = GetTime<std::chrono::microseconds>();
+    auto now = NodeClock::now();
@@ -2218 +2218 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
-        if (InactivityCheck(*pnode, now)) pnode->fDisconnect = true;
+        if (InactivityCheck(*pnode, now.time_since_epoch())) pnode->fDisconnect = true;
```

Reply:
> because time_point doesn't support being an atomic.

Duration does not either. I think it is fine to type `.load()` or `.store()`, where needed.

I guess there is no rush here, and this can be done, when all other places (non-atomic) are switched.